GRPCServiceA/cmd/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"database/sql"
+	"net"
+	"os"
+
+	_ "gopkg.in/go-sql-driver/mysql.v1"
+
+	"github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/user"
+	"github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/user/proto"
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	var logger log.Logger
+	logger = log.NewLogfmtLogger(os.Stdout)
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
+	logger = log.With(logger, "caller", log.DefaultCaller)
+	logger = log.With(logger, "service", "gRPCServiceA")
+
+	var db *sql.DB
+	db, err := sql.Open("mysql", "root:@(127.0.0.1:3306)/Users")
```

Comment:
for the next steps in Bootcamp review about using var en to each parameter in the connection string, at the moment try to let this string be in a file with constants.